File: Second Semester/Python Class/Activities/Lista de Exercícios 2/Ex_04.py
# 4. Faça um programa que peça dois números, base e expoente,
# calcule e mostre o primeiro número elevado ao segundo número.
# Não utilize a função de potência da linguagem.

import os
os.system('cls') # Limpar o Terminal .
os.system('color 3') # Mudança na cor das letras (Azul).

cont = 0


base = int(input("\n► Digite um número (base): ")) # Entrada do Usuário (base).
expoente = int(input("► Digite outro número (expoente): ")) # Entrada 2 do Usuário (expoente).
resultado = base

# Calculando a potência:
while cont < (expoente-1):
    resultado *= base
    cont += 1

print("\n■ O número %d elevado à %d = %d" %(base, expoente, resultado)) # Imprimindo resultados.

File: Second Semester/Algorithm Building and Programming/Activities/Week 3/Ex_04.c
// 4. Capture dois números inteiros e apresente o resultado da potência entre eles.
// • Se 𝑎=𝑏𝑐 então 𝑎=𝑏∗𝑏∗𝑏∗…. por “c” vezes

#include <stdio.h>
#include <stdlib.h>

int main(void){
    int base, expoente; // Número do usuário
    int resultado; // resultado da potência
    system("cls"); // Limpar.
    system("color 4"); // Cor Vermelha.

    // Recebendo dados do Usuário:
    printf("Digite uma Base: ");
    scanf("%d", &base);
    printf("Digite um Expoente: ");
    scanf("%d", &expoente);
    resultado = base;

    // Operações
    for(int i=1; i != expoente; i++) resultado = resultado * base;    

    // Saída
    printf("\n\n> %d elevado a %d = %d\n", base, expoente, resultado);
    
    getch();
    return 0;
}
